src/main/scala/fr/polytechnique/cmap/cnam/etl/extractors/acts/DcirMedicalActExtractor.scala

```diff
@@ -65,11 +60,21 @@ trait DcirActExtractor extends DcirExtractor[MedicalAct] {
 object DcirMedicalActExtractor extends DcirActExtractor  {
   override val columnName: String = ColNames.CamCode
   override val eventBuilder: EventBuilder = DcirAct
+
+  override def getInput(sources: Sources): DataFrame = sources.dcir.get.select(
+    ColNames.PatientID, ColNames.CamCode, ColNames.Date,
+    ColNames.InstitutionCode, ColNames.GHSCode, ColNames.Sector
+  )
 }
 
 
 object DcirBiologyActExtractor extends DcirActExtractor  {
   override val columnName: String = ColNames.BioCode
   override val eventBuilder: EventBuilder = BiologyDcirAct
   override def code = (row: Row) => row.getAs[Double](columnName).toString
+
+  override def getInput(sources: Sources): DataFrame = sources.dcir.get.select(
+    ColNames.PatientID, ColNames.CamCode, ColNames.BioCode, ColNames.Date,
```

Comment:
We don't need ColNames.CamCode for the BiologyActExtractor. This fix should not break any test.
```suggestion
    ColNames.PatientID, ColNames.BioCode, ColNames.Date,
```